### PR TITLE
[opus] Add Dockerfile and build.sh for decoder

### PR DESCRIPTION
**Pull request by FeliciaLim**
This adds the files needed to start fuzzing the Opus decoder.

### CODE REVIEW

File: projects/opus/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# limitations under the License.
+#
+################################################################################
+
+FROM ossfuzz/base-builder
+MAINTAINER flim@google.com
+RUN apt-get install -y make autoconf automake libtool wget
+
+RUN git clone https://git.xiph.org/opus.git
+RUN wget http://opus-codec.org/static/testvectors/opus_testvectors.tar.gz
```
Comment (oliverchang) [inline projects/opus/Dockerfile]: nit: can this be https?

File: projects/opus/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM ossfuzz/base-builder
+MAINTAINER flim@google.com
```
Comment (oliverchang) [inline projects/opus/Dockerfile]: also, you don't seem to be in the project.yaml file, so you won't get access to bugs. Would you like to be added?

**Comment by oliverchang** [comment]
And + @jmvalin  as a FYI here since they added the initial configuration.

**Comment by FeliciaLim** [comment]
Thanks @oliverchang! I've addressed your comments.

File: projects/opus/build.sh [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      -o $fuzzer.o
+    $CXX $CXXFLAGS \
+      $fuzzer.o \
+      -o $OUT/${fuzzer}_${build} \
+      -lFuzzingEngine .libs/libopus.a
+  done
+
+  # Setup the .options and test corpus zip files using the corresponding
+  # fuzzer's name
+  cp tests/$fuzzer.options $OUT/${fuzzer}_${build}.options
```
Comment (inferno-chromium) [inline projects/opus/build.sh]: Shouldn't these two lines be inside the above for.

Comment (FeliciaLim) [inline projects/opus/build.sh]: thanks for catching this! fixed now.

**Pull request merged**